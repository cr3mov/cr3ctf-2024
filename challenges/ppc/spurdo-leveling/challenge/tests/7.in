290 933158
23 43
16 32
