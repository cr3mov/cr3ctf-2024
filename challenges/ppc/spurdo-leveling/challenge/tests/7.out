88732 17745 2535
